packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -64,19 +69,47 @@ class _GestureArena {
     assert(isOpen);
     members.add(member);
   }
+
+  @override
+  String toString() {
+    final StringBuffer buffer = new StringBuffer();
+    if (members.isEmpty) {
+      buffer.write('<empty>');
+    } else {
+      buffer.write(members.map<String>((GestureArenaMember member) {
+        if (member == eagerWinner)
+          return '$member (eager winner)';
+        return '$member';
+      }).join(', '));
+    }
+    if (isOpen)
```

Comment:
This is an example of a class where TreeDescriptionMixin seems too heavy at the moment. Also it's actually private and never included in any descriptions currently, this toString was just used while I was debugging (and presumably will be used by the next person debugging this class).